Gems/Atom/TestData/TestData/Materials/Types/AutoBrick.materialtype

```diff
@@ -4,20 +4,20 @@
         "version": 3,
```

Comment:
same here in the data files - does the version need to change?

Reply:
No, this isn't used for anything yet. That's what @hershey5045 is working on.